crates/env/src/error.rs

```diff
@@ -19,10 +19,12 @@ use crate::engine::off_chain::OffChainError;
 use pallet_revive_uapi::ReturnErrorCode;
 
 /// Errors that can be encountered upon environmental interaction.
-#[derive(Debug, From, PartialEq, Eq)]
```

Comment:
`Eq` was removed as `alloy_sol_types::Error` does not derive it. Removing it does not seem to cause any errors in the code.